client/src/components/MoviesContainer.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import MoviesResults from './MoviesResults'
+import movies from './movies'
+import SearchContainer from './SearchContainer';
+import searchFilterType from './SearchFilter/searchFilterType';
+import sortType from './Sort/sortType'
+import BackToSearch from './BackToSearch'
+import MoviePage from './MoviePage'
+
+class MoviesContainer extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            list: [],
+            filter: searchFilterType.title,
+            sortType: sortType.releaseDate,
+            selectedMovie:null,
+            moviesOfTheSameGenre:[],
+            hasError:false
+        };
+    }
+
+    componentDidMount() {
+        this.setState({
+            list: this.sort(movies.data, this.state.sortType)
+        });
+    }
+
+    componentDidCatch(error, info){
+        this.setState({hasError:true});
```

Comment:
_minor_
It seems, that it is an unused piece of state. Do you somehow react to this in your UI?

Reply:
Good point, I've added 
if (this.state.hasError){
            return (
                <div>There is some error</div>
            );
        }
to MovieContainer.jsx